core/risk/margins_calculation.go

```diff
@@ -121,25 +115,80 @@ func (e *Engine) calculateMargins(m events.Margin, markPrice *num.Uint, rf types
 				svol, _ := num.UintFromDecimal(slippageVolume.Abs().Mul(e.positionFactor))
 				exitPrice, err = e.ob.GetCloseoutPrice(svol.Uint64(), types.SideBuy)
 				if err != nil {
-					noExit = true
+					longNoExit = true
 					if e.log.IsDebug() {
 						e.log.Debug("got non critical error from GetCloseoutPrice for Buy side",
 							logging.Error(err))
 					}
 				}
 			}
-			slippagePerUnit, negSlippage = num.UintZero().Delta(markPrice, exitPrice)
+			longSlippagePerUnit, _ = num.UintZero().Delta(markPrice, exitPrice)
+		}
+	}
+	// calculate margin maintenance short only if riskiest is < 0
+	// marginMaintenanceSht will be 0 by default
+	if riskiestSht.IsNegative() {
```

Comment:
I can't stop myself to read that as `riskiest shit`. Maybe not shortening the name here would be nicer.

Reply:
True that but I thought our aim is to make changes minimal 🤣 .

Reply:
I'll just rename the variable to `riskiestShit` to avoid confusion.